src/decode-nsh.c

```diff
@@ -0,0 +1,337 @@
+/* Copyright (C) 2015 Open Information Security Foundation
```

Comment:
nit: update year

Reply:
Can do - BTW this was actually written in 2015 - taken us a while to push our changes upstream :)